catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -3125,6 +3130,15 @@ private String buildIngestLog(CreateRequest createReq) {
         }
     }
 
+    private String getAttributeStringValue(Metacard mcard, String attribute) {
+        Attribute attr = mcard.getAttribute(attribute);
+        if (attr != null && attr.getValue() != null) {
+            return attr.getValue()
```

Comment:
To satisfy the access policy interface the map needs to be populated with string keys which is the only place this method is used. Thats is why it always gets the string representation regardless of the type. In practice I believe the attribute used should only ever be a string.